src/public/cards/entityProperties.ts

```diff
@@ -29,8 +29,8 @@ export class CSVEntityProperties {
     this.hideCard();
   }
 
-  setTitle(entityType: EntitiesTypes, headers: string[]) {
-    if (headers.length > 0) {
+  setTitle(entityType: EntityType, headers: string[]) {
+    if (headers.length > (entityType === EntityType.NODE ? 0 : 2)) {
```

Comment:
this was a bug when uploading edges, now it's fixed

Reply:
can we add a comment why 0 for node and 2 for edge